docs/components/docs/ExamplesList.tsx

```diff
@@ -140,6 +140,14 @@ export function Examples() {
       >
         Adds a custom logo component in the Admin UI. Builds on the Task Manager starter project.
       </Well>
+      <Well
+        grad="grad2"
+        heading="Custom Admin-UI Navigation"
```

Comment:
```suggestion
        heading="Custom Admin UI Navigation"
```

Reply:
For consistency with other navigation items:

<img width="272" alt="image" src="https://user-images.githubusercontent.com/872310/127319155-48f62ac5-5faa-4c63-8549-1b47527d8343.png">